src/main/java/com/nextstep/javaRacing/race/Race.java

```diff
@@ -0,0 +1,46 @@
+package com.nextstep.javaRacing.race;
+
+import com.nextstep.javaRacing.car.Car;
+import com.nextstep.javaRacing.view.InputView;
+import com.nextstep.javaRacing.view.ResultView;
+
+import java.util.List;
+
+public class Race {
+
+    protected final List<Car> cars;
+    protected final int turns;
+
+    protected Race(List<Car> cars, int turns) {
+        this.cars = cars;
+        this.turns = turns;
+    }
+
+    public static void start() {
+        InputView inputView = new InputView();
+        Race race = RaceFactory.prepareRace(inputView);
+        race.race();
+    }
+
+    public static void main(String[] args) {
+        Race.start();
+    }
```

Comment:
```suggestion
    public static void main(String[] args) {
        start();
    }
```

자기 자신의 메서드를 호출할 경우에는 따로 클래스를 명시하지 않아도 사용할 수 있습니다. 

추가로 코드는 읽어나가는 방향대로 배치하면 가독성을 더 높일 수 있어요. 지금과 같은 경우에는 `start()` 메서드가 아래에 위치한다면 더 자연스럽게 코드를 읽을 수 있을 거에요 👍🏼